contracts/Vault.vy

```diff
@@ -94,7 +94,8 @@ struct StrategyParams:
     totalDebt: uint256  # Total outstanding debt that Strategy has
     totalGain: uint256  # Total returns that Strategy has realized for Vault
     totalLoss: uint256  # Total losses that Strategy has realized for Vault
-
+    lossRatioLimit: uint256 # Allowed Percentage of price per share negative changes
```

Comment:
is not just losses, can be a swing in profits thats too big or unexpected

Reply:
I feel like this, coupled with your other comment https://github.com/yearn/yearn-vaults/pull/374#discussion_r639246982, points towards this being better implemented in `BaseStrategy`

Reply:
Also this comment https://github.com/yearn/yearn-vaults/pull/374#discussion_r639256575

instead of it living in another contract, just define it as invariants in `BaseStrategy`